network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/accordion_block.html

```diff
@@ -0,0 +1,26 @@
+
+
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block block_content %}
+<h2 class="tw-h3-heading">{{self.title}}</h2>
+
+	<div class="tw-row">
+        {% if self.title %}
```

Comment:
Hi @fessehaye! 

I was able to get the content to render properly within its accordion Item, I also made the title of the accordion component itself optional like requested!